services/controller/src/plz/controller/controller_impl.py

```diff
@@ -143,8 +143,9 @@ def get_logs(self, execution_id: str, since: Optional[int]) \
             -> Iterator[bytes]:
         return self.executions.get(execution_id).get_logs(since=since)
 
-    def get_output_files(self, execution_id: str) -> Iterator[bytes]:
-        return self.executions.get(execution_id).get_output_files_tarball()
+    def get_output_files(
+            self, execution_id: str, path: Optional[str]) -> Iterator[bytes]:
```

Comment:
Should this be `path: Optional[str] = None` so `path` can be omitted, or do we want to require an explicit `None`? The same thing occurs a few times below. Or, as far as I can tell, it is always used as `path if path is not None else ""`, so why not just `path: str = ""`?